src/components/common/ToolFooter.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Box, Button, Heading, Paragraph, Footer } from 'grommet';
+
+const ToolFooter = ({
+  backgroundColor,
+  headingText,
+  description,
+  buttonLabel,
+}) => (
+  <Footer direction="column" gap="none">
```

Comment:
```suggestion
  <Footer gap="none" {...rest}>
```
Using `{...rest}` is a good practice in case the consumer of the component wants to have control over the Footer component.

Reply:
Changed in the most recent push